model/src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -206,7 +206,8 @@ case class MethodConfiguration(
                    outputs: Map[String, AttributeString],
                    methodRepoMethod: MethodRepoMethod,
                    methodConfigVersion: Int = 1,
-                   deleted: Boolean = false
+                   deleted: Boolean = false,
```

Comment:
Yes - there's a possible future world where we might a) allow people to click through to deleted MCs from past submissions; b) show the edit history for an MC. These are both very maybe and for now the UI is safe to assume that anything they see is not deleted.